src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -913,4 +913,7 @@ This version used in: {2}</value>
   <data name="Install_version_0" xml:space="preserve">
     <value>Install version '{0}'</value>
   </data>
-</root>
+  <data name="AddNamedArguments" xml:space="preserve">
```

Comment:
Cyrus is advocating a new style for our resources, though it's not a bad idea. Hey @CyrusNajmabadi, could you chat with @Pilchie to figure out whether we want to make this resources style official? I'd rather not have inconsistency in the code base, so if we want this style, we should probably update it everywhere.